BD-2882: Add recommendation to use GIF instead of JavaScript (#6993)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/html_editor/creating_an_email_campaign.md

After you've selected your template, you'll see an overview of your email where you can quickly jump to the fullscreen editor to draft your email, change your sending info, and view warnings about deliverability or law compliance. 

{% alert tip %}
To add motion in an email with an accurate preview, use GIFs instead of elements that require JavaScript, as most inboxes don't support JavaScript.
{% endalert %}

![Email Variants panel for composing your email.][14]{: style="max-width:75%" }

{% alert important %}

> Braze will automatically remove HTML event handlers referenced as attributes. This does modify the HTML, and it is recommended to validate or re-check the email after it's completed. Learn more about [HTML handlers](https://www.w3schools.com/tags/ref_eventattributes.asp).

Reviewer: ```
Braze will automatically remove HTML event handlers referenced as attributes. This will modify the HTML, so it is recommended to re-check the email after it's completed. Learn more about [HTML handlers](https://www.w3schools.com/tags/ref_eventattributes.asp).
```

> Braze will automatically remove HTML event handlers referenced as attributes. This does modify the HTML, and it is recommended to validate or re-check the email after it's completed. Learn more about [HTML handlers](https://www.w3schools.com/tags/ref_eventattributes.asp).

Author: ```suggestion
Braze will automatically remove HTML event handlers referenced as attributes. This will modify the HTML, so it is recommended to re-check the email after it's completed. Learn more about [HTML handlers](https://www.w3schools.com/tags/ref_eventattributes.asp).
```
